website/docs/r/redshift_snapshot_schedule.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aws"
+page_title: "AWS: aws_redshift_snapshot_schedule"
+sidebar_current: "docs-aws-resource-redshift-snapshot-schedule"
+description: |-
+  Provides an Redshift Snapshot Schedule resource.
+---
+
+# Resource: aws_redshift_snapshot_schedule
+
+## Example Usage
+
+```hcl
+resource "aws_redshift_snapshot_schedule" "default" {
+	identifier = "tf-redshift-snapshot-schedule"
+	definitions = [
+		"rate(12 hours)",
+	]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `identifier` - (Optional, Forces new resource) The snapshot schedule identifier. If omitted, Terraform will assign a random, unique identifier.
+* `identifier_prefix` - (Optional, Forces new resource) Creates a unique
+identifier beginning with the specified prefix. Conflicts with `identifier`.
+* `description` - (Optional) The description of the snapshot schedule.
+* `definitions` - (Optional) The definition of the snapshot schedule. The definition is made up of schedule expressions, for example "cron(30 12 *)" or "rate(12 hours)".
```

Comment:
Nit: we might want to wrap these two examples in backticks since Markdown is interpreting the * for italics

```suggestion
* `definitions` - (Optional) The definition of the snapshot schedule. The definition is made up of schedule expressions, for example `cron(30 12 *)` or `rate(12 hours)`.
```

Reply:
Thanks, fixed it.